types/audit-details.d.ts

```diff
@@ -218,12 +218,23 @@ declare global {
        */
       export interface SourceLocationValue {
         type: 'source-location';
-        /** urls from the network are always valid urls. otherwise, urls come from either a comment or header, and may not be well-formed. */
+        /** A "url" representing the source file. May not be a valid URL, see `urlProvider`. */
         url: string;
-        /** 'network' when the url is the actual, observed resource url. 'comment' when the url comes from a sourceMapURL comment or X-SourceMap header */
+        /**
+         * - `network` when the url is the actual, observed resource url. This is always a valid URL.
+         * - `comment` when the url comes from a sourceURL comment. This could be anything, really.
+         */
         urlProvider: 'network' | 'comment';
+        /** Zero-indexed. */
         line: number;
         column: number;
+        /** The original file location from the source map. */
+        original?: {
+          /** The relevant file from the map's `sources` array. If missing, could not associate mapping with a specific file ("unmapped" in other tools). */
+          file?: string;
```

Comment:
is this a useless state? Are the line/col numbers still useful if we can't say which file the line/col numbers are in?